public_api/src/index.ts

```diff
@@ -34,6 +36,8 @@ import {
 } from "mcbe-addon-ipc";
 
 export * from "./registry_types.js";
+export * as NetworkLinks from "./network_links/network_links.js"
```

Comment:
This should be `networkLinks` not `NetworkLinks` as module aliases should be `camelCase`

Reply:
To me that would indicate that it's an instance of a class or a field for example, when in this case its functionality a namespace

Reply:
When you use a module alias in an import you use `camelCase`, it should be the same for an export